tools/deploy-static.sh

```diff
@@ -14,10 +14,10 @@ DIVISION_LINE=`awk '/####*/ {print NR}' config/redirects.txt | sed -n 3p`
 head -n $DIVISION_LINE config/redirects.txt > build/_redirects
 
 # normalize all Cloudflare Style Redirects to end with /
-# sed -r 's/^(\/[^ ]+\/[^\/ ]+) /\1\/ /g' -i build/_redirects
+sed -r 's/^(\/[^ ]+\/[^\/ ]+) /\1\/ /g' -i build/_redirects
 
 # duplicate every Cloudflare Style Redirect ending with / to also end without /
-# sed -rn 'p; s/^(\/[^ ]+)\/ /\1 /p' -i build/_redirects
+sed -rn 'p; s/^(\/[^ ]+)\/ /\1 /p' -i build/_redirects
```

Comment:
Don't enable these, I'm removing them in a subsequent PR.

Reply:
If that's the case you can take up these changes in your PR. I am fine with it.

Reply:
Can close this one